assets/js/theme-edit.js

```diff
@@ -23,21 +23,41 @@
 	}
 
 	function appleNewsSelectInit() {
-		// Only show fonts on Macs since they're system fonts
-		if ( appleNewsSupportsMacFeatures() ) {
-			$( '.select2.standard' ).select2();
-			$( '.select2.font' ).select2({
-				templateResult: appleNewsFontSelectTemplate,
-				templateSelection: appleNewsFontSelectTemplate
-			});
-		} else {
-			$( '.select2' ).select2();
-			$( 'span.select2' ).after( // phpcs:ignore WordPressVIPMinimum.JS.HTMLExecutingFunctions.after
-				$( '<div>' )
-					.addClass( 'font-notice' )
-					.text( appleNewsThemeEdit.fontNotice )
-			)
-		}
+		$( '.select2.standard' ).select2();
+		$( '.select2.font' ).select2({
+			templateResult: appleNewsFontSelectTemplate,
+			templateSelection: appleNewsFontSelectTemplate
+		}).on('select2:select', async function (e) {
+			// Check if font preview is available.
+			var selectedFont = e.params.data.text;
```

Comment:
Should we use `const` or `let` here? And in the next few lines?

Reply:
I used `var` to remain consistent with the rest of the file, but just noticed I used `let` in some cases. So I will update these to `const`!

Reply:
I believe this is _not_ transpiled JS, so I'd recommend having them stay as `var` - the `let` was probably an oversight

Reply:
It was my understanding these were pretty universally supported now, is that not the case?